service/src/main/java/bio/terra/workspace/amalgam/landingzone/azure/LandingZoneApiDispatch.java

```diff
@@ -243,21 +249,26 @@ private ApiAzureLandingZoneResult toApiAzureLandingZoneResult(
         .landingZone(azureLandingZone);
   }
 
-  /**
-   * TODO (https://broadworkbench.atlassian.net/browse/TOAZ-221) This method is an initial
-   * implementation that must be revised, and likely be refactored once WSM stores the LZ id in the
-   * azure context. The initial assumption is that the cardinality of 1:1 between the cloud context
-   * and the LZ is enforced in the create operation, therefore more than one LZ per azure context is
-   * not allowed.
-   */
-  public UUID getLandingZoneId(AzureCloudContext azureCloudContext) {
-    var landingZoneTarget = MapperUtils.LandingZoneTargetMapper.from(azureCloudContext);
```

Comment:
Glad to resolve this TODO.

Are we able to remove the method `MapperUtils.LandingZoneTargetMapper.from(CloudContext)`?

Reply:
Yes. I removed it. Thanks.